managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -361,6 +369,18 @@ public void operationComplete(ManagedCursorInfo info, Stat stat) {
                 cursorLedgerStat = stat;
                 lastActive = info.getLastActive() != 0 ? info.getLastActive() : lastActive;
 
+
+                Map<String, String> recoveredCursorProperties = Collections.emptyMap();
+                if (info.getCursorPropertiesCount() > 0) {
+                    // Recover properties map
+                    recoveredCursorProperties = Maps.newHashMap();
```

Comment:
nit: in 4 lines here (and in some other places in the change) we have two ways of creating new map (`Collections.emptyMap()` and `Maps.newHashMap()`).
While nothing is wrong with either one having both requires reviewer to think about motivation for having two, differences, etc. Do we really need both?

Reply:
Exactly my point ;) Can't guess that without looking at the library code or docs.
"ImmutableMap.of()" can express this in a more visible way.